e2e/__tests__/__snapshots__/stackTraceSourceMapsWithCoverage.test.ts.snap

```diff
@@ -0,0 +1,20 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`processes stack traces and code frames with source maps with coverage 1`] = `
+FAIL __tests__/fails.ts
+  ✕ fails
+
+  ● fails
+
+    This did not work!
+
+      11 | 
+      12 | export function error() {
+    > 13 |   throw new Error('This did not work!');
+         |         ^
+      14 | }
+      15 | 
+
+      at Object.error (lib.ts:13:9)
+      at Object.<anonymous> (__tests__/fails.ts:10:3)
+`;
```

Comment:
For posterity (if we ever come back to this and wonder why we pass sourcemap into Babel), this is the error if running this test without the other changes in this PR

![image](https://user-images.githubusercontent.com/1404810/80144169-52e89780-85ae-11ea-804d-43bde1cd4f2d.png)